apps/channels/models.py

```diff
@@ -11,9 +11,14 @@
 from apps.utils.models import BaseModel
 from apps.web.meta import absolute_url
 
+WEB = "web"
+TELEGRAM = "telegram"
+WHATSAPP = "whatsapp"
+
 
 class ExperimentChannel(BaseModel):
-    PLATFORM = (("telegram", "telegram"), ("web", "web"), ("whatsapp", "whatsapp"))
+    PLATFORM = ((TELEGRAM, TELEGRAM), (WEB, WEB), (WHATSAPP, WHATSAPP))
```

Comment:
```suggestion
    PLATFORM = ((TELEGRAM, "Telegram"), (WEB, "Web"), (WHATSAPP, "WhatsApp"))
```

is there a reason you chose not to add the capitalized display names?